### ISSUE TITLE
cannot connect to host

### ISSUE THREAD

**Issue by Lucas03lescure** [ElevenPaths/Eternalblue-Doublepulsar-Metasploit#85]
i runned eternalblue and configured everything corectly but this still pops up no matter what

Started reverse TCP handler on 192.168.9.188:4444
[*] 192.168.10.188:4444 - Connecting to target for exploitation.
[-] 192.168.10.188:4444 - Rex::ConnectionRefused: The connection was refused by the remote host (192.168.10.188:4444).
[*] Exploit completed, but no session was created.

**Comment by PLEASEFORGETME2** [comment]
@Lucas03lescure Of course, you tried to do it on the port 4444, which is not open !
Scan for open ports with nmap then truc with them...

**Comment by Lucas03lescure** [comment]
oof i've forgoten about this thread lol, thanks for the advice though it makes sense now though if I do recall correctly the port 4444 was indeed open

**Comment by PLEASEFORGETME2** [comment]
Yes I know it's an old thread, and I wanted to see if it was patched... And
yes it is. So I guess your Windows version was patched against this exploit

Le mar. 20 août 2019 à 09:13, Lucas03lescure <notifications@github.com> a
écrit :

**Comment by GetRektBoy724** [comment]
> @Lucas03lescure Of course, you tried to do it on the port 4444, which is not open !
> Scan for open ports with nmap then try with them...

Kinda funny tho lmao
